app/controllers/unattended_controller.rb

```diff
@@ -67,6 +67,12 @@ def pxe_config
   # Using alias_method causes test failures as iPXE method is unknown in an empty DB
   def gPXE; iPXE; end
 
+  protected
+
+  def require_ssl?
```

Comment:
Sure, it "works", but previously it would still require SSL if the user was accessing the spoof view (from host#show > Templates > Review).  This change will allow that to happen over a non-SSL connection, which shouldn't be the case.